lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js

```diff
@@ -1,6 +1,6 @@
 const React = require('react');
 
-const title = "Whoo! You're Signed Up.";
+const title = "This is how you do it.";
```

Comment:
Didn't we _just_ switch this? https://github.com/DoSomething/phoenix/pull/6811 Is it cause the slide order showing is changing?